### DISCUSSION TITLE
How to run yolov8 model with ort?

### DISCUSSION THREAD

**Discussion by Fancyflame** [pykeio/ort#115] [Q&A] [answered]
Hi there,
I'm a newbie in ML and have two-year experience in Rust. I'm trying to run an onnx file trained by the Yolov8 model to detect objects. It works well in Python, but I don't know how to run it in ort. I have read the documentation and the gpt example, but still don't know what to do and what every step in the example means. Does anybody have succeeded or have tutorials? Thanks a lot!

Now I
- have the `yolov8s.onnx` file
- compiled ort successfully, but haven't tested the example because I cannot download gpt model for unknown network reasons.

**Comment by decahedron1** [comment, marked as answer]
There is a YOLOv8 example on the v2 branch, maybe this can help point you in the right direction.

https://github.com/pykeio/ort/blob/v2/examples/yolov8/examples/yolov8.rs

**Reply by Fancyflame** [reply]
Thanks, I'll take a try :)

**Reply by Fancyflame** [reply]
I have figured out how to run yolo model, much thanks to you and this example. But I wonder if this file's input and output arguments have some documentation or standard? I checked `yolov8m.onnx` on an [onnx viewer](https://netron.app/) can only see these without further description.
<img width="350" alt="image" src="https://github.com/pykeio/ort/assets/31250625/74ee6b89-6644-4b1d-85a9-b4dbc4f16860">
